features/site/home/CommunityShowcase.tsx

```
@@ -0,0 +1,202 @@
+import { ThemeTint } from '@tamagui/logo'
+import { memo, useState } from 'react'
+import { Button, Card, GetThemeValueForKey, Paragraph, Text, XGroup, XStack, YStack } from 'tamagui'
```

Biome (1.9.4)

[error] Some named imports are only used as types.

This import is only used as a type.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Add inline type keywords.


(lint/style/useImportType)